context/app/static/js/components/Detail/ProvTableTile/ProvTableTile.jsx

```diff
@@ -1,20 +1,21 @@
 import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
 
 import useEntityData from 'hooks/useEntityData';
 import DetailContext from '../context';
 import EntityTile from '../EntityTile';
 import { DownIcon } from './style';
 
 function ProvTableTile(props) {
-  const { uuid, entity_type, id, isCurrentEntity, isNotSibling } = props;
+  const { uuid, entity_type, id, isCurrentEntity, isSampleSibling, isFirstTile } = props;
   const { elasticsearchEndpoint } = useContext(DetailContext);
 
   // mapped fields are not included in ancestor object
   const entityData = useEntityData(uuid, elasticsearchEndpoint);
 
   return (
     <>
-      {entity_type === 'Sample' && isNotSibling && <DownIcon />}
+      {!isFirstTile && ((entity_type === 'Sample' && !isSampleSibling) || entity_type === 'Dataset') && <DownIcon />}
```

Comment:
```suggestion
      {!isFirstTile && !isSampleSibling && <DownIcon />}
```
`isSampleSibling` already checks that that tyle is `Sample` so can this be simplified?

Reply:
We don't want Donors to display the icon and while we don't expect more than one donor, I'd prefer to be overly strict.

Reply:
I could use `!isFirstTile && !isSampleSibling && entity_type !== 'Donor`, I believe.